src/mod2c_core/nocpout.c

```diff
@@ -530,38 +529,6 @@ fprintf(stderr, "Notice: ARTIFICIAL_CELL models that would require thread specif
 		Sprintf(buf, "#define %s %s%s\n", STR(q), STR(q), suffix);
 		Lappendstr(defs_list, buf);
 	}
-
-	Lappendstr(defs_list, "/* external NEURON variables */\n");
-	SYMLISTITER {
-		s = SYM(q);
-		if (s->nrntype & NRNEXTRN) {
-			if (strcmp(s->name, "dt") == 0) { continue; }
-			if (strcmp(s->name, "t") == 0) { continue; }
-			if (s->subtype & ARRAY) {
-				Sprintf(buf, "extern double* %s;\n", s->name);
```

Comment:
we can't remove these externs. NMODL has extern keyword where you can reference variable from another mod file. Here you are removing those extern declarations (which will be compilation error).

Reply:
ok, good to know, we can discuss that, as those can be defined in coreneuron too: celsius, t and dt are defined in coreneuron for instance.